packages/server/src/workers/subscription.ts

```diff
@@ -290,10 +290,25 @@ async function getSubscriptions(resource: Resource): Promise<Subscription[]> {
       },
     ],
   });
-  const inMemorySubscriptionsStr = await getRedis().get(`medplum:subscriptions:r4:project:${project}`);
-  if (inMemorySubscriptionsStr) {
-    const inMemorySubscriptions = JSON.parse(inMemorySubscriptionsStr) as Subscription[];
-    subscriptions.push(...inMemorySubscriptions);
+  const redisOnlySubRefStrs = await getRedis().smembers(`medplum:subscriptions:r4:project:${project}:active`);
+  const redisOnlySubStrs = await getRedis().mget(redisOnlySubRefStrs);
+  if (redisOnlySubStrs.length) {
+    const arrLen = redisOnlySubStrs.length;
+    let subArrStr = '[';
```

Comment:
Not sure, but I'd wager that using `.join(',')` would be faster than for loop here

```ts
const subArrStr = '[' + redisOnlySubStrs.filter(Boolean).join(',') + ']';
```